app/schemas/models/exam.schema.js

```diff
@@ -0,0 +1,59 @@
+const c = require('./../schemas')
+
+const ExamSchema = c.object({
+  title: 'Exam',
+  description: 'CodeCombat exam with limited levels and complete time',
+})
+
+_.extend(ExamSchema.properties, {
+  title: {
+    type: 'string',
+    description: 'Title of the exam',
+  },
+  description: {
+    type: 'string',
+    description: 'Description of the exam',
+  },
+  clan: c.objectId({ links: [{ rel: 'db', href: '/db/clan/{($)}' }] }),
```

Comment:
why do we need clan? Does it refer to mto clan? We do hope to do some ai-league at some point for the exam in future.

Reply:
clan for easy get a group of users. for example this time to include all mto users. and in the futures can easy add exams to the classrooms/apiclients/...